### PR TITLE
Lighting mappings

### PR DESCRIPTION
**Pull request by Grayray75**
Some lighting mappings I need for my Phosphor port

### CODE REVIEW

**Review by thecatcore — APPROVED** [review]

File: mappings/net/minecraft/world/chunk/Chunk.mapping [hunk 1]
```diff
@@ -1,6 +1,9 @@
 CLASS net/minecraft/class_1196 net/minecraft/world/chunk/Chunk
 	FIELD field_12911 EMPTY Lnet/minecraft/class_1197;
+	FIELD field_12912 unloaded Z
 	FIELD field_4725 surfaceCache [I
+	FIELD field_4726 columSkyLightOutdated [Z
+		COMMENT A list of colums which need their skylight to be checked
```
Comment (BleachDev) [inline mappings/net/minecraft/world/chunk/Chunk.mapping:6]:
```suggestion
	FIELD field_4726 columnSkyLightOutdated [Z
		COMMENT A list of columns which need their skylight to be checked
```

File: mappings/net/minecraft/world/chunk/Chunk.mapping [hunk 2]
```diff
@@ -129,11 +133,13 @@ CLASS net/minecraft/class_1196 net/minecraft/world/chunk/Chunk
...
 		ARG 3 z
-	METHOD method_3909 (IIII)V
+	METHOD method_3909 calculateSkyLightForRegion (IIII)V
 		ARG 1 x
 		ARG 2 z
+		ARG 3 beginY
+		ARG 4 endY
 	METHOD method_3910 setModified ()V
-	METHOD method_3911 (II)V
+	METHOD method_3911 setColumLightOutdated (II)V
```
Comment (BleachDev) [inline mappings/net/minecraft/world/chunk/Chunk.mapping:142]:
```suggestion
	METHOD method_3911 setColumnLightOutdated (II)V
```

File: mappings/net/minecraft/world/chunk/Chunk.mapping [hunk 3]
```diff
@@ -162,8 +170,8 @@ CLASS net/minecraft/class_1196 net/minecraft/world/chunk/Chunk
 		ARG 3 z
 	METHOD method_6533 populateBlockEntities (Z)V
 		ARG 1 runningBehind
-	METHOD method_6534 (Z)V
-		ARG 1 isClient
+	METHOD method_6534 recheckSkyLightGaps (Z)V
+		ARG 1 onlyOneColum
```
Comment (BleachDev) [inline mappings/net/minecraft/world/chunk/Chunk.mapping:174]:
```suggestion
		ARG 1 onlyOneColumn
```

File: mappings/net/minecraft/world/chunk/Chunk.mapping (hunk 1 above)
Comment (Grayray75) [reply mappings/net/minecraft/world/chunk/Chunk.mapping]: thanks for pointing that out

**Review by BleachDev — APPROVED** [review]

**Pull request merged**